### DISCUSSION TITLE
Documentation References

### DISCUSSION THREAD

**Discussion by pedronauck** [FuelLabs/fuels-ts#381] [Ideas]
The idea here is to talk about some references and ideas to create our SDK documentation.

IMHO, one of the websites I really like in terms of design/organization is Wagmi

- https://wagmi.sh/docs/getting-started

What do you think about talking about this here?

**Comment by luizstacio** [comment]
I like the Solana docs: https://docs.solana.com/introduction
It provides a nice overview and guide

**Reply by camiinthisthang** [reply]
Hey @pedronauck - good call on bringing the convo into GitHub. Want to clarify that we’re talking about the “top level” docs here? I know there are diff teams, each working on their own gitbook so just want to make sure we’re talking about the same thing. Anywho, these are some of the best docs I’ve come accross:
 
The [Nextjs docs](https://nextjs.org/docs/getting-started) are world class imo, and they get lots of compliments on them
[Motion One](https://motion.dev/guides/feature-comparison) also has really good docs. Specifically I like that they have a Quickstart, Example section, and feature comparison section
The [MarkDoc](https://markdoc.io/docs/getting-started) docs are also really nice

**Comment by LuizAsFight** [comment]
This one is also good:

https://mui.com/material-ui/getting-started/installation/

- components section
  - component working in the page
  - code to make it (option to see full code instead of just the component portion)
- migration
  - we can have a migration section explaining differences between solidity and sway, with guides on how to migrate


IMO a great search mechanism is a **must** to have
